modules/music_module.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../classes/musicAlbum'
+require_relative '../classes/genre'
+require 'date'
+
+module MusicModule
+    def take_album_details
+        puts 'Please add a Music Album with details below: '
+        item_property_selection
+        print 'Music Album Source: '
+        @m_source = gets.chomp
```

Comment:
From the given diagram We do not have to implement a source class would you remove the source attribute